pom.xml

```diff
@@ -116,7 +116,7 @@
 		<dependency.log4j2.version>2.20.0</dependency.log4j2.version>
 		<dependency.spotbugs-annotations.version>4.7.3</dependency.spotbugs-annotations.version>
 		<dependency.saxon-he.version>12.4</dependency.saxon-he.version>
-		<dependency.xmlresolver.version>6.0.2</dependency.xmlresolver.version>
+		<dependency.xmlresolver.version>6.0.4</dependency.xmlresolver.version>
```

Comment:
I am going to park this along with Log4J updates until I discuss with Dave the cascading version check strategy from m-j -> liboscal-java -> oscal-cli enforced by Maven [in the upstream m-j dependency](https://github.com/usnistgov/metaschema-java/blob/28468999d802e69273df7e725d183c132e2b15d8/pom.xml#L151). Oddly, there is no dependabot update there yet, so waiting on this.